docs/upgrade.md

```diff
@@ -31,3 +31,15 @@ NOTE: rolling-update does not yet perform a real rolling update - it just shuts
  there will be downtime [Issue #37](https://github.com/kubernetes/kops/issues/37)
 We have implemented a new feature that does drain and validate nodes.  This feature is experimental, and you can use the new feature by setting `export KOPS_FEATURE_FLAGS="+DrainAndValidateRollingUpdate"`.
 
+### Terraform Users
+
+* `kops edit cluster $NAME`
+* set the KubernetesVersion to the target version (e.g. `v1.3.5`)
+* NOTE: The next 3 steps must all be ran in the same directory
+* `kops update cluster $NAME --target=terraform`
+* `terraform plan`
```

Comment:
@justinsb @chrislovecnm I honestly don't know how these files ended up here. Like Chris said, I had some issues with Git rebase. Any suggestions on how to fix this?